presto-main/src/main/java/com/facebook/presto/operator/AggregationOperator.java

```diff
@@ -203,12 +245,24 @@ public Page getOutput()
 
     private static class Aggregator
     {
-        private final Accumulator aggregation;
+        private final Accumulator accumulator;
         private final Step step;
         private final int intermediateChannel;
 
         private Aggregator(AccumulatorFactory accumulatorFactory, Step step)
         {
+            this(accumulatorFactory, step, ImmutableList.of(), ImmutableList.of(), ImmutableList.of(), Optional.empty());
+        }
+
+        private Aggregator(
```

Comment:
such mechanical refactorings could go to a separate commit, so it's easier to recognize core changes